pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -72,22 +94,48 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Replica Quiescence" sources={storeSources}>
+    <LineGraph
+      title="Replica Quiescence"
+      sources={storeSources}
+      tooltip={
+        `The total number of replicas and quiesced replicas.
+        Quiesced replicas are ones that have not been accessed in a while.`
+      }
+    >
       <Axis label="replicas">
         <Metric name="cr.store.replicas" title="Replicas" />
         <Metric name="cr.store.replicas.quiescent" title="Quiescent" />
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Range Operations" sources={storeSources}>
+    <LineGraph title="Range Operations" sources={storeSources} tooltip={`Total ranges with split, add, or remove operations.`}>
       <Axis label="ranges">
         <Metric name="cr.store.range.splits" title="Splits" nonNegativeRate />
         <Metric name="cr.store.range.adds" title="Adds" nonNegativeRate />
         <Metric name="cr.store.range.removes" title="Removes" nonNegativeRate />
       </Axis>
     </LineGraph>,
 
-    <LineGraph title="Snapshots" sources={storeSources}>
+    <LineGraph
+      title="Snapshots" 
+      sources={storeSources} 
+      tooltip={(
+        <div>
+          When a node is far behind the log file for a range, the cluster can
+          send it a snapshot of the range and it can start following the log from there.
+          <dl>
+            <dt>Generated</dt>
+            <dd>The number of snapshots created per second</dd>
+            <dt>Applied (Raft-initiated)</dt>
+            <dd>The number of snapshots applied to nodes per second that were initiated within Raft</dd>
+            <dt>Applied (Preemptive)</dt>
+            <dd>The number of snapshots applied to nodes per second that were anticipated ahead of time</dd>
+            <dt>Reserved</dt>
+            <dd>The number of slots reserved per second for incoming snapshots that will be sent to a node</dd>
```

Comment:
I think you'll want to add `${tooltipSelection}` to each of these descriptions to make things perfectly clear.

- The number of snapshots created `${tooltipSelection}` per second
- The number of snapshots applied `${tooltipSelection}` per second...
- The number of slots reserved `${tooltipSelection}` per second...